src/steps/account.ts

```diff
@@ -33,7 +34,16 @@ export async function fetchAccountDetails({
     config: instance.config,
   });
 
-  const oktaSupportInfo = await apiClient.getSupportInfo();
+  let oktaSupportInfo: OrgOktaSupportSettingsObj | undefined = undefined;
+  try {
+    oktaSupportInfo = await apiClient.getSupportInfo();
+  } catch (err) {
+    logger.info(`Unable to query Okta Support Info due to ERROR:  `, err);
```

Comment:
Minor - I think we should make this consistent with the other ways that we have been including errors in our logs. Makes searching more consistent.

```suggestion
    logger.info({ err }, 'Unable to query Okta Support Info due to error.');
```